ui/StatusQ/src/StatusQ/Layout/+qt6/StatusSectionLayoutLandscape.qml

```diff
@@ -182,20 +182,24 @@ SplitView {
             color: Theme.palette.statusAppLayout.rightPanelBackgroundColor
         }
         contentItem: Item {
-            Item {
+            LayoutItemProxy {
                 id: headerBackgroundSlot
                 anchors.top: parent.top
                 // Needed cause I see a gap otherwise
                 anchors.topMargin: -3
-                width: visible ? parent.width : 0
-                height: visible ? childrenRect.height : 0
-                visible: (!!headerBackground)
+                width: parent.width
+                target: root.headerBackground
+                z: -1
```

Comment:
Is that needed? Usually manually manipulating the `z` order means something is wrong there :)